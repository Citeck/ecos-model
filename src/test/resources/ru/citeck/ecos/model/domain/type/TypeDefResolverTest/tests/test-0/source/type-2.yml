---
id: type-2

sourceType: ECOS_MODEL

aspects:
  - ref: emodel/aspect@aspect-1
