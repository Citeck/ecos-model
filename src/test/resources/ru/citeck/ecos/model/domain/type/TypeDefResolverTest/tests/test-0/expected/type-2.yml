---
id: type-2
name:
  en: type-2

sourceType: ECOS_MODEL
sourceId: emodel/type-2

metaRecord: emodel/type-2@

aspects:
  - ref: emodel/aspect@aspect-1

inheritCreateVariants: true

model:
  attributes:
    - id: aspect-1:aspect-1-att
  systemAttributes:
    - id: aspect-1:aspect-1-system-att
